boardgame-ui/src/pages/LoginPage.vue

```diff
@@ -48,6 +107,49 @@ export default {
         console.log(error);
         this.error = true;
```

Comment:
future improvement (not in scope of this PR): store status code and display reason of connection failure (invalid credentials vs account not yet validated, maybe others?)